schema.capabilities.json

```diff
@@ -72,6 +72,13 @@
         "subtotals": {
             "description": "Specifies the subtotal customizations applied in the customizeQuery method",
             "$ref": "#/definitions/subtotals"
+        },
+        "migration": {
+            "$ref": "#/definitions/migration"        
+        },
+        "keepAllMetadataColumns": {
+            "type": "boolean",
+            "description": "Indicates that visual is going to receive all metadata columns, no matter what the active projections are"
```

Comment:
Is it a necessary sentence "no matter what the active projections are"? can it be removed

Reply:
IMO it is necessary since this is the core difference between having this flag set as true and undefined (as the current behaviour). We need to differentiate the two scenarios and the received columns in each

Reply:
I mean i think its  good enough and clear if we only keep the first part of the sentence "Indicates that visual is going to receive all metadata columns" 
without saying "no matter to active projections..." since we indicated if it's true then "that visual is going to receive all metadata columns" as it specified in first sentence
Gennady after all, if you still see that its better with the whole sentence you wrote then you can keep it and resolve the comment